osquery/examples/extension_group_example/README.MD

```diff
@@ -0,0 +1,9 @@
+All the extensions made with the **add_osquery_extension_ex()** CMake function will be automatically bundled into a single executable.
```

Comment:
Instead of this inline `README` approach, can we add a section to the `/docs/wiki` with similar content?

Reply:
Yes, that's better! I moved everything inside the wiki and deleted my README file